mercantile/__init__.py

```diff
@@ -845,31 +833,28 @@ def x_minmax(zoom):
 
     Returns
     -------
-    Tuple of (min, max) integers
-
-    """
-
-    if zoom != int(zoom) or int(zoom) < 0:
-        raise InvalidZoomError("zoom must be a positive integer")
-
-    return (0, 2 ** zoom - 1)
-
-
-def y_minmax(zoom):
-    """Returns the min and max y tile coordinates for a specific zoom level
+    minimum : int
+        Minimum tile coordinate (note: always 0).
+    maximum : int
+        Maximum tile coordinate (2 ** zoom - 1).
 
-    Parameters
-    ----------
-    zoom : int
-        The web mercator zoom level.
+    Raises
+    ------
+    InvalidZoomError
+        If zoom level is not a positive integer.
 
-    Returns
-    -------
-    Tuple of (min, max) integers
+    Examples
+    --------
+    >>> minmax(1)
+    (0, 1)
+    >>> minmax(-1)
+    Traceback (most recent call last):
+    ...
+    InvalidZoomError: zoom must be a positive integer
 
     """
 
-    if zoom != int(zoom) or int(zoom) < 0:
+    if not isinstance(zoom, int) or int(zoom) < 0:
```

Comment:
Let's require `int` and disallow `float`.

Reply:
I was following the other functions in mercantile with the `v != int(v)` pattern, from what I've seen this is used throughout mercantile. Should we keep it for consistency? Or change all of the int checks then?

Reply:
Let's leave the checks in other functions as they are in case we have users who are passing in floats and counting on existing functions to round them. Hyrum's Law :upside_down_face: